user.css

```
@@ -788,4 +788,136 @@ body.spotui-custom-bar-on #spotui-custom-bar {
+    border: 1px solid var(--spotui-accent, #ff8c42) !important;
+    border-radius: 0 !important;
+    color: var(--spotui-accent, #ff8c42) !important;
+}
+
+body.spotui-spotify-enabled .main-card-card .marketplace-installButton:hover {
+    background: var(--spotui-accent, #ff8c42) !important;
+    color: #000 !important;
+}
+
+body.spotui-spotify-enabled .main-card-card .marketplace-installButton svg {
+    fill: currentColor !important;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)